src/Dobble/DeckValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Marmelab\Dobble;
+
+abstract class DeckValidator
+{
+    public static function validate($deck)
+    {
+        self::validateDeckIsNotEmpty($deck);
```

Comment:
the problem with each problem throwing one exception is that, if there is more than one problem, you will discover it after the first problem is fixed... So if `validate()` can throw, I think the other functions should return null or a validation problem.